DeDRM_plugin/adobekey.py

```diff
@@ -548,27 +553,27 @@ def cli_main():
 
 def gui_main():
     try:
-        import Tkinter
-        import Tkconstants
-        import tkMessageBox
+        import six.moves.tkinter
+        import six.moves.tkinter_constants
+        import six.moves.tkinter_messagebox
```

Comment:
```suggestion
        from six.moves import tkinter, tkinter_constants, tkinter_messagebox
```
Doing it this way will require fewer changes now and fewer changes when we drop legacy Python and six.  It will also improve runtime performance.